preprocess.sh

```diff
@@ -6,5 +6,6 @@ mkdir data
 unzip data_compressed/codex-l.zip -d data
 unzip data_compressed/codex-m.zip -d data
 unzip data_compressed/codex-s.zip -d data
+unzip data_compressed/codex-test.zip -d data
```

Comment:
as a test dataset already has a semantic meaning in our context (test vs. training dataset) I would prefer to name it codex-xxs or something else